Doc/library/pathlib.rst

```diff
@@ -1055,3 +1071,41 @@ call fails (for example because the path doesn't exist):
       'Text file contents'
 
    .. versionadded:: 3.5
+
+Correspondence to tools in the os package
+-----------------------------------------
+
+If you're more familiar with :mod:`os.path` module, here's a correspondence
+table on how the same things may be accomplished with pathlib.
+
+.. note::
```

Comment:
I see in this file there is no blank line after `.. note::`, but this role does not apply to the rest of the doc.  Maybe @brettcannon could tell us what is the policy.  Another question: what to do in case the policy is to have a blank line?  Do we stay consistent with the file or with the general policy?